src/prefect_server/graphql/schema/flows.graphql

```diff
@@ -151,7 +154,14 @@ input set_flow_group_labels_input {
   labels: [String!]
 }
 
-	input set_flow_group_schedule_input {
+input set_flow_group_run_config_input {
+  "The ID of the flow group to update"
+  flow_group_id: UUID!
+  "The serialized run config to associate with the flow group"
+  run_config: JSON
```

Comment:
Confirming that `run_config` is optional because we want to allow `None` to be provided - is that accurate?

Reply:
Correct.